api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -124,8 +124,9 @@ paths:
                 $ref: "#/definitions/Amount"
                 description: Set the coin amount that will be transferred to the smartContract.
               fee:
-                $ref: "#/definitions/Amount"
-                description: Set the fee amount that will be given to the block creator.
+                type : string
+                description: Set the fee amount (in massa) that will be given to the block creator.
+                default: "0"
```

Comment:
here we fix the fee parameter, before it was a Amount in nonamassa but the endpoint expect to have the value in MAS. It was not failed the tests because 1000 nano mas or mas, it passes.

also, we fix the documentation